stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/BaseMessageItemViewHolder.kt

```diff
@@ -25,14 +25,14 @@ public abstract class BaseMessageItemViewHolder<T : MessageListItem>(
      * Workaround to allow a downcast of the MessageListItem to T
      */
     @Suppress("UNCHECKED_CAST")
-    internal fun bindListItem(messageListItem: MessageListItem, isThread: Boolean, diff: MessageListItemPayloadDiff? = null) {
```

Comment:
It's really nice to see all of these parameters gone. Looks like we've ended up with both a more correct and a cleaner solution than what we originally had.